blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -49,7 +48,8 @@ export class ReplayButton extends Component {
             replaying: true,
         });
 
-        runApi.replayRun(this.props.latestRun)
+        runApi
+            .replayRun(this.props.latestRun)
```

Comment:
ick?

Reply:
IMO better. Also, the point isn't pleasing people, it's consistency and mechanical transformation.

Reply:
love it

Reply:
It looks like the rule is "multiple chained methods each go on their own line" which I am fine with.

Reply:
yeah, i can live with this just fine